ui/raidboss/data/07-dt/raid/r8s.txt

```diff
@@ -1,9 +1,328 @@
+341.7 "--shadow 4--" #Ability { id: ["A3E0", "A3E1"], source: "Moonlit Shadow" }
+343.7 "Terrestrial Rage" Ability { id: "A3BE", source: "Howling Blade" }
+349.3 "Heavensearth" Ability { id: "A3C0", source: "Howling Blade" }
+349.3 "Suspended Stone" #Ability { id: "A3BF", source: "Howling Blade" }
+350.7 "Moonbeam's Bite 1" #Ability { id: ["A3C2", "A3C3"], source: "Moonlit Shadow" }
+352.7 "Moonbeam's Bite 2" #Ability { id: ["A3C2", "A3C3"], source: "Moonlit Shadow" }
+354.7 "Moonbeam's Bite 3" #Ability { id: ["A3C2", "A3C3"], source: "Moonlit Shadow" }
+356.7 "Moonbeam's Bite 4" #Ability { id: ["A3C2", "A3C3"], source: "Moonlit Shadow" }
+357.7 "Heavensearth" Ability { id: "A3C0", source: "Howling Blade" }
+357.7 "Suspended Stone" #Ability { id: "A3BF", source: "Howling Blade" }
+357.7 "--middle--" Ability { id: "A38F", source: "Howling Blade" }
+361.9 "Weal of Stone" Ability { id: "A792", source: "Wolf of Stone" }
+366.8 "Stonefang/Windfang" Ability { id: ["A39E", "A39D", "A3A1", "A3A2"], source: "Howling Blade" }
+375.8 "Tracking Tremors" Ability { id: "A3B9", source: "Howling Blade" }
+376.7 "Tracking Tremors 1" #Ability { id: "A3BB", source: "Howling Blade" }
+377.7 "Tracking Tremors 2" #Ability { id: "A3BB", source: "Howling Blade" }
+378.7 "Tracking Tremors 3" #Ability { id: "A3BB", source: "Howling Blade" }
+379.7 "Tracking Tremors 4" #Ability { id: "A3BB", source: "Howling Blade" }
+380.7 "Tracking Tremors 5" #Ability { id: "A3BB", source: "Howling Blade" }
+381.8 "Tracking Tremors 6" #Ability { id: "A3BB", source: "Howling Blade" }
+382.8 "Tracking Tremors 7" #Ability { id: "A3BB", source: "Howling Blade" }
+383.8 "Tracking Tremors 8" #Ability { id: "A3BB", source: "Howling Blade" }
+389.7 "Extraplanar Pursuit" Ability { id: "A74F", source: "Howling Blade" }
+404.5 "Extraplanar Pursuit" Ability { id: "A74F", source: "Howling Blade" }
+405.2 "--middle--" #Ability { id: "A38F", source: "Howling Blade" }
+405.2 "--untargetable--"
+405.5 "--sync--" StartsUsing { id: "A3F1", source: "Howling Blade" } window 10,10
+409.2 "Extraplanar Feast (Enrage)" Ability { id: "A3F1", source: "Howling Blade" }
+
+# Phase 2
+505.0 "--middle--" Ability { id: "A38F", source: "Howling Blade" } window 140,0
+506.0 "--sync--" Ability { id: "A82D", source: "Howling Blade" } window 510,0
+550.3 "--targetable--"
+562.3 "Quake III" Ability { id: "A45A", source: "Howling Blade" }
+574.3 "Ultraviolent Ray 1" Ability { id: "A45C", source: "Howling Blade" }
+574.3 "Gleaming Beam" Ability { id: "A45E", source: "Gleaming Fang" }
+585.3 "Twinbite" Ability { id: "A4CD", source: "Howling Blade" }
+597.4 "Hero's Blow" Ability { id: ["A460", "A462"], source: "Howling Blade" }
+607.4 "Ultraviolent Ray 2" Ability { id: "A45C", source: "Howling Blade" }
+607.4 "Gleaming Beam" Ability { id: "A45E", source: "Gleaming Fang" }
+618.4 "Quake III" Ability { id: "A45A", source: "Howling Blade" }
+630.6 "Mooncleaver" Ability { id: "A465", source: "Howling Blade" }
```

Comment:
```suggestion
629.9 "Mooncleaver (castbar)" StartsUsing { id: "A465", source: "Howling Blade" } duration 3.7
```
Opinion only, feel free to ignore. (Please change over all the Mooncleaver uses if you implement this one.)

Reply:
I've changed it to use duration and StartsUsing. Also included Starcleaver.